lib/ffmpeg/qsv/encoder.py

```diff
@@ -158,6 +162,10 @@ def encode(self):
         assert result == md5ref, "r2r md5 mismatch"
         # delete encoded file after each iteration
         get_media()._purge_test_artifact(self.encoded)
+    elif vars(self).get("forced_idr", None) is not None:
+      self.check_bitrate()
+      self.check_metrics()
+      self.check_output()
```

Comment:
This `elif` is not needed since the `else` does the same thing.

Reply:
Another note, the order here is not good since `check_metrics` can store the PSNR result in baseline if user specifies `--rebase`, but if `check_output` fails next, the rebased metric is useless.

Reply:
Either way, just remove this `elif` block and let the `else` do the work in the current correct order.